src/bin/pgcopydb/pgcmd.c

```diff
@@ -802,13 +801,9 @@ pg_restore_roles(PostgresPaths *pgPaths,
 				continue;
 			}
 
-			char createRole[BUFSIZE] = { 0 };
-
-			sformat(createRole, sizeof(createRole), "CREATE ROLE %s", roleName);
-
-			log_info("%s", createRole);
+			log_info("%s", currentLine);
 
-			if (!pgsql_execute(&pgsql, createRole))
+			if (!pgsql_execute(&pgsql, currentLine))
```

Comment:
Again, this doesn't look to me like it was done by `cppcheck`, what's the story behind that change?

Reply:
It started with the `shadowed variable name` and then I realized an opporunity to simplify by completely removing the variable.